crates/catalog-unity/src/datafusion.rs

```diff
@@ -163,17 +166,40 @@ impl UnitySchemaProvider {
                 .collect(),
             ListTableSummariesResponse::Error(_) => vec![],
         };
+        let token_cache = Cache::builder()
+            .max_capacity(1000)
```

Comment:
Probably won't hurt us, as credentials are not large, but 1000 seems like a lot for a single client?